### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Sergej161995**
[DEMO LINK](https://Sergej161995.github.io/layout_catalog/)
[TEST REPORT LINK](https://Sergej161995.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Your links are not working, try to fix them or ask for help in fe_chat

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,394 @@
...
+       >
+        <img src="./images/logo.png" alt="catalog-logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+             class="nav__link is-active"
+             >
```
Comment (AlisaVlasova) [inline src/index.html:38]:
```suggestion
            <a href="#apple" class="nav__link is-active">
```

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,394 @@
...
+            class="nav__link">
+            Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
```
Comment (AlisaVlasova) [inline src/index.html:61]:
```suggestion
            <a
              href="#laptops"
              class="nav__link"
              data-qa="nav-hover"
            >
```

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,394 @@
...
+             >
+             Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung"
+            class="nav__link"
+            >
+            Samsung
+            </a>
```
Comment (AlisaVlasova) [inline src/index.html:47]:
```suggestion
            >
              Samsung
            </a>
```

File: src/index.html [hunk 4]
```diff
@@ -5,8 +5,394 @@
...
+    <main class="catalog-main">
+
+      <section class="cards">
+
+        <div class="card cards__position" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="iMac A1419"
+            >
```
Comment (AlisaVlasova) [inline src/index.html:110]:
```suggestion
    <main class="catalog-main">
      <section class="cards">
        <div class="card cards__position" data-qa="card">
          <img
              class="card__img"
              src="./images/imac.jpeg"
              alt="iMac A1419"
            >
```

File: src/styles/blocks/catalog-header.scss [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__logo {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav {
```
Comment (AlisaVlasova) [inline src/styles/blocks/catalog-header.scss:13]: move nav into separate file

File: src/styles/blocks/catalog-main.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      margin-bottom: $cards-margin-b;
+    }
+
+    .card:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    .card:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
```
Comment (AlisaVlasova) [inline src/styles/blocks/catalog-main.scss:23]:
```suggestion
  .cards {
    max-width: getWidht($card-width, $cards-margin-r);
    @extend %flex-center;
    flex-wrap: wrap;
    padding: 50px 0;

    &__position {
      margin-right: $cards-margin-r;
      margin-bottom: $cards-margin-b;

        &:nth-child(4n) {
          margin-right: 0;
        }

        &:nth-last-child(-n + 4) {
          margin-bottom: 0;
        }
      }
```

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,391 @@
...
+    </header>
+
+    <main class="catalog-main">
+      <section class="cards">
+        <div class="card cards__position" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="iMac A1419"
+            >
```
Comment (TarasHoliuk) [inline src/index.html:107]:
```suggestion
          <img
            class="card__img"
            src="./images/imac.jpeg"
            alt="iMac A1419"
          >
```

File: src/index.html [hunk 8]
```diff
@@ -5,8 +5,391 @@
...
+
+          <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
```
Comment (TarasHoliuk) [inline src/index.html:111]:
```suggestion
          <h2 class="card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
```

File: src/index.html [hunk 9]
```diff
@@ -5,8 +5,391 @@
...
+            <div class="card__price">Price:</div>
+            <div class="card__cost">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+            data-qa="card-hover"
+            >
+              Buy
+          </a>
```
Comment (TarasHoliuk) [inline src/index.html:136]:
```suggestion
          <a
            class="card__buy"
            href="#"
            data-qa="card-hover"
          >
            Buy
          </a>
```

File: src/styles/blocks/star.scss [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    background-position: center;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+@for $starCount from 1 through 5 {
+  .stars--#{starCount} .stars__star:nth-child(-n+#{$starCount}) {
```
Comment (TarasHoliuk) [inline src/styles/blocks/star.scss:21]: It seems like it doesn't work:

image

That's also the reason tests fail
Please, fix it

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Everything looks good except for 1 thing (check comment)

File: src/index.html (hunk 7 above)
Comment (vlad-demchuk) [reply src/index.html:107]: check the previous comments

**Review by vlad-demchuk — COMMENTED** [review]
It seems you didn't push changes. check the previous comments.
stars still aren't working
image

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
/

**Pull request closed without merging**